config/wdio.android.app.conf.js

```diff
@@ -0,0 +1,23 @@
+const { config } = require('./wdio.shared.conf');
+
+config.cucumberOpts.tagExpression = `${config.cucumberOpts.tagExpression} and not @ios`;
+// ============
+// Capabilities
+// ============
+// For all capabilities please check
+// http://appium.io/docs/en/writing-running-appium/caps/#general-capabilities
+config.capabilities = [
```

Comment:
Leave an empty line between the comment and the start of this block

Reply:
done